bundles/binding/org.openhab.binding.snmp/README.md

```diff
@@ -63,18 +65,18 @@ where:
 Here are some examples of valid binding configuration strings:
 
 ```
-snmp="<[192.168.2.111:public:.1.3.6.1.2.1.2.2.1.10.10:10000]"
+snmp="<[192.168.2.111:v1:public:.1.3.6.1.2.1.2.2.1.10.10:10000]"
 snmp=">[OFF:192.168.2.111:private:.1.3.6.1.4.1.4526.11.16.1.1.1.3.1.2:2]"
-snmp="<[192.168.2.111:public:.1.3.6.1.2.1.2.2.1.10.10:0]"
+snmp="<[192.168.2.111:v2c:public:.1.3.6.1.2.1.2.2.1.10.10:0]"
 ```
 
 ## Examples
 
 items/snmpdemo.items
 
 ```
-Number Switch_POEState2  "PoE WiFi State  [%s]"  { snmp="<[192.168.2.111:public:.1.3.6.1.4.1.4526.11.16.1.1.1.6.1.2:10000]" }
-Switch Switch_POEEnable2 "PoE WiFi Enable [%s]"  { snmp="<[192.168.2.111:public:.1.3.6.1.4.1.4526.11.16.1.1.1.3.1.2:10000] >[OFF:192.168.2.111:private:.1.3.6.1.4.1.4526.11.16.1.1.1.3.1.2:2] >[ON:192.168.2.111:private:.1.3.6.1.4.1.4526.11.16.1.1.1.3.1.2:1]" }
+Number Switch_POEState2  "PoE WiFi State  [%s]"  { snmp="<[192.168.2.111:v1:public:.1.3.6.1.4.1.4526.11.16.1.1.1.6.1.2:10000]" }
+Switch Switch_POEEnable2 "PoE WiFi Enable [%s]"  { snmp="<[192.168.2.111:pv1:ublic:.1.3.6.1.4.1.4526.11.16.1.1.1.3.1.2:10000] >[OFF:192.168.2.111:private:.1.3.6.1.4.1.4526.11.16.1.1.1.3.1.2:2] >[ON:192.168.2.111:private:.1.3.6.1.4.1.4526.11.16.1.1.1.3.1.2:1]" }
```

Comment:
Presumably this is a typo.

Should `pv1:ublic` be `v1:public` here?

Reply:
yes, it was. Thanks for the correction

Reply:
Are you planning to push a fix for it, then?